Core/HLE/sceSas.cpp

```diff
@@ -363,8 +363,9 @@ static u32 sceSasSetVoice(u32 core, int voiceNum, u32 vagAddr, int size, int loo
 	v.type = VOICETYPE_VAG;
 	v.vagAddr = vagAddr;  // Real VAG header is 0x30 bytes behind the vagAddr
 	v.vagSize = size;
-	v.loop = loop ? true : false;
-	v.ChangedParams(vagAddr == prevVagAddr);
+	v.loop = loop != 0;
+	v.playing = true;
```

Comment:
I think we should still care about `on`?  You can key on or off a voice before or after setting the VAG, I'm pretty sure tests do show that already...

-[Unknown]

Reply:
Oh yeah, I suppose so. I'll add that in.